app/controllers/admin/groups_controller.rb

```diff
@@ -47,16 +47,12 @@ def update
     params[:adults] = false if params[:adults].nil?
     params[:healthy_volunteers] = false if params[:healthy_volunteers].nil?
 
+    @group.subgroups = params[:tags].to_s.split(',').map do |subgroup|
+      @group.subgroups.build(name: subgroup)
+    end
+    @group.save
```

Comment:
This double save is a bit odd. Might want to refactor later.

Reply:
Agreed!